bdk_chain/src/keychain.rs

```diff
@@ -33,11 +38,15 @@ pub use keychain_txout_index::*;
     )
 )]
 #[must_use]
-pub struct DerivationAdditions<K>(BTreeMap<K, u32>);
+pub struct DerivationAdditions<K> {
+    /// The updates (if any) to the last-derived index per keychain.
+    pub last_derived: BTreeMap<K, u32>,
```

Comment:
I didn't see what was wrong with the way it was before. It could use a method `as_inner` or something that returns an immutable ref to the `BTreeMap`.